paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/settings/PlaygroundSettings.kt

```diff
@@ -136,38 +169,69 @@ internal class PlaygroundSettings private constructor(
         }
     }
 
+    @Serializable
+    private class SerializableSettings(
+        val configurationData: PlaygroundConfigurationData,
+        val settings: Map<String, String>,
+    )
+
     companion object {
         private const val sharedPreferencesName = "PlaygroundSettings"
         private const val sharedPreferencesKey = "json"
 
         fun createFromDefaults(): PlaygroundSettings {
+            val defaultConfigurationData = PlaygroundConfigurationData()
+
             val settings = allSettingDefinitions.associateWith { settingDefinition ->
-                MutableStateFlow(settingDefinition.defaultValue)
+                val defaultValue = settingDefinition.playgroundConfigurable()?.convertToValue(defaultConfigurationData)
```

Comment:
This is a little odd. Now anything that conforms to this kinda has 2 default values.